crates/junction-api/src/xds/http.rs

```diff
@@ -70,9 +87,45 @@ impl Route {
                         .with_field_index("virtual_hosts", idx);
                 };
 
+                let vhost_ratelimit: xds_local_ratelimit::LocalRateLimit = vhost
+                    .typed_per_filter_config
+                    .get("envoy.filters.http.local_ratelimit")
+                    .map(|any| {
+                        protobuf::Any::to_msg(any).or(Err(Error::new_static(
+                            "Failed to parse local rate limit config",
+                        )
+                        .with_field("envoy.filters.http.local.ratelimit")))
+                    })
+                    .transpose()
+                    .and_then(|r| r.ok_or(Error::new_static("missing local rate limit config")))
```

Comment:
Not sure if this is a TODO or not, but this probably shouldn't error - we should be able to send xDS with no rate limit config. 

(I think yesterday you said it was a TODO, but here's a note just in case)